traefik/traefik.yml

```diff
@@ -21,15 +15,12 @@ providers:
     swarmModeRefreshSeconds: 15s
     exposedByDefault: false
   file:
-    directory: dynamic/
+    directory: dynamic
 
 api:
   dashboard: true
   debug: true
 
-pilot:
```

Comment:
Thought the deprecation of Traefik Pilot was gonna be a problem for below plugins, but these two features are apparently separate.